src/repos/tree/index.ts

```diff
@@ -61,7 +61,7 @@ class RepositoryTreeProvider implements TreeDataProvider<TreeItem> {
   getChildren(element?: TreeItem): ProviderResult<TreeItem[]> {
     if (!element && authStore.isSignedIn && store.repos.length > 0) {
       return store.repos
-        .sort((a, b) => a.name.localeCompare(b.name))
+        .slice().sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
This change makes sense, though I'm curious if there was an actual issue that this fixed?

Reply:
It was throwing an exception on this line, it was clear in the debug console: the error message had a suggestion about how to fix it

Reply:
You're right, I did further testing last night and the real root cause was the exception from `base64-to-uint8array`. Do you want me to revert this change?